pkg/config/configuration.go

```diff
@@ -385,6 +387,10 @@ func (fp *FrontendParameters) SetDefaultValues() {
 	if fp.QueryResultMaxsize == 0 {
 		fp.QueryResultMaxsize = 100
 	}
+
+	if fp.AutoIncrCacheSize == 0 {
+		fp.AutoIncrCacheSize = 3000
```

Comment:
I think default can be 30000. or 20000
3000 is too small.   tidb is 30000.
@fengttt